notebooks/Gaslim_draw.ipynb

```diff
@@ -134,17 +134,16 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "catalog_names = [os.path.join('/Users/remy/Desktop/LSST_Project/GalSim/examples/', \n",
-    "                            'data/COSMOS_25.2_training_sample', \n",
-    "                            'real_galaxy_catalog_25.2.fits'),\n",
-    "                 os.path.join('/Users/remy/Desktop/LSST_Project/GalSim/examples/', \n",
-    "                            'data/COSMOS_25.2_training_sample', \n",
-    "                            'real_galaxy_catalog_25.2_fits.fits')]\n",
-    "galsim_catalog = galsim.COSMOSCatalog(catalog_names[0])\n",
+    "# Change this to your own path to galsim 'real_galaxy_catalog_25.2.fits'\n",
+    "galsim path = '/Users/remy/Desktop/LSST_Project/GalSim/examples/data/COSMOS_25.2_training_sample'\n",
+    "                            \n",
```

Comment:
We might leave this for now but we could switch to using environment variables or the input YAML file for these kind of path in order to avoid leaving personal details in the main repo.